_wallets/cobowallet.md

```diff
@@ -0,0 +1,65 @@
+---
+# This file is licensed under the MIT License (MIT) available on
+# http://opensource.org/licenses/MIT.
+
+id: Cobo
+title: "Cobo Wallet"
+titleshort: "Cobo"
+compat: "mobile android ios"
+level: 3
+platform:
+  - mobile:
+    name: mobile
+    os:
+      - name: android
+        text: "walletcobo"
+        link: "https://play.google.com/store/apps/details?id=cobo.wallet"
+        source: "https://github.com/cobowallet"
```

Comment:
A direct link to the repository containing the wallet source code is missing here.